OrangeHRMTests/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.Data;
+using OrangeHRMTests.PageObjects.Elements;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class BasePage : WebDriverFactory
+    {
+        private MyWebElement CreatedEmployeeListFirstPosition = new MyWebElement(By.XPath("//div[@role='listbox']/div[1]/span"));
+        private MyWebElement EmployeeNameHintedTextBoxElement = new MyWebElement(By.XPath("//label[@class='oxd-label'][text()='Employee Name']//ancestor::div[1]" +
+            "//following-sibling::div[1]//input"));
+
+        public void CreateEmployee()
```

Comment:
BasePage should contain only methods that interact with page
CreateEmployee/CreateUser and some below them are methods that doing smth rather interact with page
if this methods are common for several tests in different test suites - move them to BaseTest, not BasePage

Reply:
Fixed